dmc2gym/wrappers.py

```diff
@@ -161,11 +182,30 @@ def reset(self):
         obs = self._get_obs(time_step)
         return obs
 
-    def render(self, mode='rgb_array', height=None, width=None, camera_id=0):
-        assert mode == 'rgb_array', 'only support rgb_array mode, given %s' % mode
-        height = height or self._height
-        width = width or self._width
-        camera_id = camera_id or self._camera_id
-        return self._env.physics.render(
-            height=height, width=width, camera_id=camera_id
+    def render(self, mode="rgb_array", height=None, width=None, camera_id=0, **kwargs):
+        img = self._env.physics.render(
+            height=self.render_kwargs["height"] if height is None else height,
```

Comment:
I think easier to read if you did these default argument replacements on their own lines before calculating `img`.

Reply:
Good suggestion

Reply:
True!